plugins/pooling_function.cc

```diff
@@ -51,7 +52,10 @@ PoolingFunction(MldbServer * owner,
 {
     functionConfig = config.params.convert<PoolingFunctionConfig>();
 
-    set<string> validAggs = {"avg", "min", "max", "sum"};
+    SqlQueryFunctionConfig fnConfig;
+    fnConfig.query.stm.reset(new SelectStatement());
+
+    set<Utf8String> validAggs = {"avg", "min", "max", "sum"};
```

Comment:
Ask @mailletf.  I simply ported the existing functionality to a new execution engine.